geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/rebalance/model/PartitionedRegionLoadModel.java

```diff
@@ -468,6 +507,41 @@ public Move findBestTarget(Bucket bucket, boolean checkIPAddress) {
     return bestMove;
   }
 
+  String getRedundancyZone(InternalDistributedMember memberID) {
+    assert (partitionedRegion != null);
```

Comment:
This is a situation only a test should hit if it doesn't supply a partitioned region. That said, I changed it to a throw from an assert based on your. We know in advance this can be a problem so i am testing for it.

Reply:
I would argue that if a test is not supplying a partitioned region, then the test hasn't been written properly and throwing the NPE on the next line should be fine. Changing it to any other kind of exception doesn't really solve anything, just makes it fail in a different way.

Reply:
Fair.

Reply:
Changed.